OpenMEEG/tools/meshes_to_vtp.cpp

```diff
@@ -49,50 +49,59 @@ int main( int argc, char **argv) {
 
     print_version(argv[0]);
 
-    command_usage("Convert meshes into a single VTK/VTP file.");
+    command_usage("Convert meshes OR geometry into a single VTK/VTP file.");
+    const char * geom_filename;
     const char * input[7];
     const char * name[7];
     const char * output;
-    input[0] = command_option("-i1", (const char *) NULL, "Input Mesh");
-    input[1] = command_option("-i2", (const char *) NULL, "Input Mesh");
-    input[2] = command_option("-i3", (const char *) NULL, "Input Mesh");
-    input[3] = command_option("-i4", (const char *) NULL, "Input Mesh");
-    input[4] = command_option("-i5", (const char *) NULL, "Input Mesh");
-    input[5] = command_option("-i6", (const char *) NULL, "Input Mesh");
-    input[6] = command_option("-i7", (const char *) NULL, "Input Mesh");
-    name[0]  = command_option("-n1", (const char *) "1", "Mesh Name");
-    name[1]  = command_option("-n2", (const char *) "2", "Mesh Name");
-    name[2]  = command_option("-n3", (const char *) "3", "Mesh Name");
-    name[3]  = command_option("-n4", (const char *) "4", "Mesh Name");
-    name[4]  = command_option("-n5", (const char *) "5", "Mesh Name");
-    name[5]  = command_option("-n6", (const char *) "6", "Mesh Name");
-    name[6]  = command_option("-n7", (const char *) "7", "Mesh Name");
+    geom_filename = command_option("-g", (const char *) NULL, "Input geometry");
```

Comment:
nothing changed for meshes.
But I added a way so that the user can (instead of providing each mesh separately), specify the geometry file (where meshes are already there and named) to convert it into a single Vtk/vtp file